Content/1.1 - Transaction Review/transactionReview.json

```diff
@@ -18,7 +18,7 @@
 	"transactionReview_poolUnits": "Pool Units",
 	"transactionReview_slideToSign": "Slide to Sign",
 	"transactionReview_xrdAmount": "%s XRD",
-	"transactionReview_insufficientBalance": "Insufficient Balance",
+	"transactionReview_insufficientBalance": "Not enough XRD for transaction fee",
```

Comment:
Umair created a [bug](https://radixdlt.atlassian.net/browse/ABW-3377) for this. Android doesn't have this issue it is because they are using the value defined under [Customize Network Fees](https://github.com/radixdlt/apps-localization-src/blob/main/Content/1.1.1%20-%20Customize%20Network%20Fees/customizeNetworkFees.json#L32) on both [Transaction Review](https://github.com/radixdlt/babylon-wallet-android/blob/main/app/src/main/java/com/babylon/wallet/android/presentation/transaction/composables/FeesSheet.kt#L206) & [Customize Fees](https://github.com/radixdlt/babylon-wallet-android/blob/main/app/src/main/java/com/babylon/wallet/android/presentation/transaction/composables/NetworkFeeContent.kt#L99).

Reply:
but Android should switch to the correct key `transactionReview_insufficientBalance`, also to have both platforms aligned.

Reply:
definitely! Was just pointing out to make it clear why this change wouldn't break anything on Android.